task_manager/task_manager.hh

```diff
@@ -0,0 +1,112 @@
+#pragma once
+
+#include "replica/database.hh"
+
+class task_manager {
+public:
+    static const size_t modules_count = 2;
+    enum class module {
+        repair = 0,
+        compaction
+    };
+    class running_tasks {
```

Comment:
I'm not sure why we need this class.
Can't we simply sum some `containers.size()`?